### DISCUSSION TITLE
silly.. but navbar with x-axis scroll?

### DISCUSSION THREAD

**Discussion by Hyp3rionM4x** [twbs/bootstrap#35036] [Ideas]
Hi, so i came across this "simple" idea. I'm also pretty sure i saw this in one of your sample templates (the blog i think). When you lower your resolution on devices et and your navbar goes under x-pixels etc, is it possible to add in so that the user will be able to scroll with the x-axis instead of adding a drop-down menu? Both is pretty great and would be awesome if you could add this small peace of CSS-Code to your documentation on the website because right now i have to go to the template all the time to check out how you guys do it.

I know, seems simple enought but still, would just be awesome if you were to add it... just an idea, hope you will do it with next release/update!!!!!!!!!!!!!!!!  :D :D :D :D :D :D :D :D :D

**Comment by ghost** [comment]
Are you referring to a floating menu for smaller devices and fixed-at-the-top-navbar for bigger devices?

**Comment by coliff** [comment]
I think you are referring to the `navbar-nav-scroll` class which adds the following CSS:

```css
.navbar-expand .navbar-nav-scroll {
  overflow: visible;
}
```
You can see this in action on the Bootstrap Blog navbar menu.

![image_from_ios](https://user-images.githubusercontent.com/1212885/134375690-31879c8c-a0b7-4a06-8d64-501b0a02df26.png)
